users/config.rst

```diff
@@ -1309,6 +1309,13 @@ Defaults Element
             <untrusted>false</untrusted>
             <remoteGUIPort>0</remoteGUIPort>
         </device>
+	<ignores>
+            <line>!foo2</line>
+            <line>// comment</line>
+            <line>(?d).DS_Store</line>
+            <line>*2</line>
+            <line>qu*</line>
+        </ignores>
```

Comment:
An indentation step too much?

Reply:
I don't think so, the rest of the code example is also indented with four spaces per level.  I did sneak another tab in there though, which is now fixed.